go-controller/pkg/network-attach-def-controller/network_manager.go

```diff
@@ -214,7 +215,13 @@ func (nm *networkManagerImpl) syncAll() error {
 	start := time.Now()
 	klog.Infof("%s: syncing all networks", nm.name)
 	for _, network := range validNetworks {
-		if err := nm.sync(network.GetNetworkName()); err != nil {
+		if err := nm.sync(network.GetNetworkName()); errors.Is(err, ErrNetworkControllerTopologyNotManaged) {
```

Comment:
wow learnt something new here, so from code perspective, its a localnet network of type ovn-k8s-cni-overlay with  no subnets meaning its pure l2 usecase ? OR something else does IPAM for those VM pods and they talk to the VLAN outside?

so this means we don't create any NAD controller for such networks? hence ErrNetworkControllerTopologyNotManaged ?

Reply:
No, we have one, but running in the ovnkube-node.

Reply:
Hmm why do we create one in the node side? Was it for dpu use case?

Reply:
what does that controller do on the node side? is it for the DPU case?